a11y-meta-display-guide/2.0/draft/guidelines/index.html

```diff
@@ -2462,41 +2462,47 @@ <h2>Implementations</h2>
 			possibly volunteer an implementation, please put
 			your company name, your name and your contact information.
 		</div>
+		-->
+
 
 		<p>These guidelines provide a general framework and make
 			suggestions on the display of accessibility
 			metadata. It is not a normative description of what must be
 			done. It is instructive to provide examples
 			of implementations from the community. </p>
 
-		<p>Linked below are static pages that show real-life
-			implementations. We have captured these examples from
+		<p>We have created a webpage that captures examples from
```

Comment:
Best to avoid pronouns and be specific. I'd probably change this to:

The Publishing Community Group has created a web page that captures examples from web sites belonging to organizations that have agreed to showcase the the work they have done to display accessibility metadata.